bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/config/OpenWeatherMapAPIConfiguration.java

```diff
@@ -16,7 +16,6 @@
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
-import org.openhab.binding.openweathermap.internal.handler.OpenWeatherMapAPIHandler;
```

Comment:
It was probably used by the Javadoc link, so you now need to provide the namespace explicitly three lines below:
```java
* The {@link OpenWeatherMapAPIConfiguration} is the class used to match the {@link org.openhab.binding.openweathermap.internal.handler.OpenWeatherMapAPIHandler}s
````

I didn't check in IDE, but otherwise you can probably not navigate/mouse-over correctly.

Reply:
Yep, this PR is partial, it is a draft. Need to fix those

Reply:
Fixed. Not sure if this is the best, but atleast the warnings have disapeared. Too bad the javadoc is not taken into account for compilation warnings.